src/OpenTelemetry/Trace/BatchSpanExportProcessorOptions.cs

```diff
@@ -0,0 +1,91 @@
+// <copyright file="BatchSpanExportProcessorOptions.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Diagnostics;
+using System.Security;
+using OpenTelemetry.Internal;
+
+namespace OpenTelemetry.Trace
+{
+    public class BatchSpanExportProcessorOptions : BatchExportProcessorOptions<Activity>
+    {
+        internal const string MaxQueueSizeEnvVarKey = "OTEL_BSP_MAX_QUEUE_SIZE";
+
+        internal const string MaxExportBatchSizeEnvVarKey = "OTEL_BSP_MAX_EXPORT_BATCH_SIZE";
+
+        internal const string ExporterTimeoutEnvVarKey = "OTEL_BSP_EXPORT_TIMEOUT";
+
+        internal const string ScheduledDelayEnvVarKey = "OTEL_BSP_SCHEDULE_DELAY";
+
+        public BatchSpanExportProcessorOptions()
```

Comment:
Could you update public api as well.. (We temporarily disabled it for metrics work, so CI is not checking it.)

Reply:
Sure. Is there any tooling to help with it? How to test it (e.g. how to run the check which is disabled)?

Reply:
@pellared If you want to re-enable the check locally make this block...

https://github.com/open-telemetry/opentelemetry-dotnet/blob/c9c78691cec293af20972e55b661b14b126cb55a/build/Common.prod.props#L4-L15

...like this...

```xml
  <ItemGroup>
    <PackageReference Include="MinVer" Version="$(MinVerPkgVer)" Condition="'$(IntegrationBuild)' != 'true'">
      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
      <PrivateAssets>all</PrivateAssets>
    </PackageReference>
    <PackageReference Include="Microsoft.CodeAnalysis.PublicApiAnalyzers" Version="$(MicrosoftCodeAnalysisAnalyzersPkgVer)" Condition=" $(OS) == 'Windows_NT'">
      <PrivateAssets>All</PrivateAssets>
    </PackageReference>
  </ItemGroup>
```

And then you should get the warnings and code fixes to resolve them.

Reply:
Thank you.

Addressed 018b6a49 

Shouldn't we remove the `.publicApi/net46` and `.publicApi/net452` directories as these .NET Framework versions are no longer supported?

Reply:
@cijothomas @CodeBlanch  see ⬆️